### PR TITLE
Wysiwyg

### PR DESCRIPTION
**Pull request by soupette**
Add full screen editor and preview

### CODE REVIEW

File: packages/strapi-helper-plugin/lib/src/components/InputFileDetails/index.js [hunk 1]
```diff
@@ -8,18 +8,18 @@ import React from 'react';
...
 import { FormattedMessage } from 'react-intl';
 import { get, startsWith } from 'lodash';
-import cn from 'classnames';
 
 import styles from './styles.scss';
 
 function InputFileDetails(props) {
   if (props.number === 0 && props.multiple) {
-    return <div />;
+    console.log('fycj');
```
Comment (Aurelsicoko) [inline packages/strapi-helper-plugin/lib/src/components/InputFileDetails/index.js]: Log

File: packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      return styles.editorBlockquote;
+    case 'code-block':
+      return styles.editorCodeBlock;
+    default:
+      return null;
+  }
+}
+
+export function getBlockContent(style) {
+  switch (style) {
```
Comment (Aurelsicoko) [inline packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js]: Just asking why some values are uppercased (IMG, BOLD, ...) and some others are lowercased (blockquote, code-block)? We should keep the same format (lowercased) whatever the value is.

File: packages/strapi-helper-plugin/package.json [hunk 3]
```diff
@@ -48,6 +48,8 @@
     "copy-webpack-plugin": "^4.3.1",
     "cross-env": "^5.0.5",
     "css-loader": "^0.28.5",
+    "draft-js": "^0.10.5",
+    "draft-js-import-markdown": "^1.2.1",
```
Comment (Aurelsicoko) [inline packages/strapi-helper-plugin/package.json]: This dependency is never required...

File: packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js]: Agree with you but the format should be uppercased

Comment (soupette) [reply packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js]: Fixed in uppercase

**Comment by oeddyo** [comment]
looking forward to this feature! Is it coming soon?

And thanks for the hard work guys!!!

**Comment by dzuncoi** [comment]
Any updates on this PR guys? Really excited to see this feature

**Comment by soupette** [comment]
We're still testing it, but it should merged sometime next week

**Comment by Aurelsicoko** [comment]
@nainsworthrv I hope we will merge it this week, let me double-check everything because we made some updates recently

**Comment by nainsworthrv** [comment]
@Aurelsicoko awesome- thanks for the hard work loving strapi!

**Comment by Aurelsicoko** [comment]
FYI, we still have some weird behaviours due to DraftJS. We are working on it, the merge is delayed to the next week.

**Comment by oeddyo** [comment]
Well can't complain anything...

Just want to let you know that I really feel Strapi to be the best supported open source project. I've used many other tools, and personally involved with some (twitter's tool called scalding), but I have to say this is the best core team and community I've ever seen. I feel great potential for this tool and how a general problem Strapi is solving for many many other startups.

Great job guys! Thanks for all the efforts! Going to buy a t-shirt for every member of my team soon :D

**Comment by Aurelsicoko** [comment]
LGTM 👍

**Comment by zhuang17** [comment]
Hi @soupette, @lauriejim, @Aurelsicoko 
I saw this plugin was released in the new [version 3.0.0-alpha.12](https://github.com/strapi/strapi/releases/tag/v3.0.0-alpha.12).
Great work on this.

A quick question, 
how can i enable the `WYSIWYG` editor now using the new version?

I tried update the strapi version, but I didn't find any advanced `WYSIWYG`  setting for Text field in Admin UI.
I also tried install the `strapi-helper-plugins` will i guess maybe needed to enable `WYSIWYG`, but i can't find it in Plugins Marketplace (`http://localhost:1337/admin/install-plugin`)

so anyone can help me on how to enable this, or let me know if i missed anything ?

**Comment by lauriejim** [comment]
Install new version of strapi `npm i strapi@alpha -g` and create new project.
If you have an existing project, we will write a migration guide.

WYSIWYG is not a plugin. it's config inside content manager.

**Comment by raulriera** [comment]
> WYSIWYG is not a plugin. it's config inside content manager.

This line helped me, when you create a `text` field there is an option in the advanced tab to make it WYSIWYG

**Comment by JamyGolden** [comment]
@raulriera thanks for the clarification!

**Pull request merged**